05_syntax_lab/01.py

```diff
@@ -1,5 +1,10 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+
+print "Please choose Number"
+x = int(raw_input())
+
+for number in range(1,10) :
+    print "choose agian"
+    num = int(raw_input())
+    x = max(num,x)
+
+print "Highes number is" ,x
```

Comment:
Hi and a late welcome to the exercises!

You could eliminate the duplicate call to ```raw_input``.